feature/bgp/policybase/otg_tests/statement_insertion_removal/README.md

```diff
@@ -0,0 +1,1130 @@
+                      "match-community-set": {
+                        "config": {
+                          "community-set": "Comm_100_7",
+                          "match-set-options": "ANY"
+                        }
+                      }
+                    }
+                  },
+                  "openconfig-routing-policy:actions": {
+                    "config": {
+                      "policy-result": "NEXT_STATEMENT"
+                    }
+                  }
+                },
+                {
+                  "name": "Stmnt_8",
+                  "openconfig-routing-policy:config": {
+                    "name": "Stmnt_8"
+                  },
+                  "openconfig-routing-policy:conditions": {
+                    "openconfig-bgp-policy:bgp-conditions": {
+                      "match-community-set": {
+                        "config": {
+                          "community-set": "Comm_100_8",
+                          "match-set-options": "ANY"
+                        }
+                      }
+                    }
+                  },
+                  "openconfig-routing-policy:actions": {
+                    "config": {
+                      "policy-result": "NEXT_STATEMENT"
+                    }
+                  }
+                },
+                {
+                  "name": "Stmnt_10",
+                  "openconfig-routing-policy:config": {
+                    "name": "Stmnt_10"
+                  },
+                  "openconfig-routing-policy:conditions": {
+                    "openconfig-bgp-policy:bgp-conditions": {
+                      "match-community-set": {
+                        "config": {
+                          "community-set": "Comm_100_10",
+                          "match-set-options": "ANY"
+                        }
+                      }
+                    }
+                  },
+                  "openconfig-routing-policy:actions": {
+                    "config": {
+                      "policy-result": "NEXT_STATEMENT"
+                    }
+                  }
+                },
+                {
+                  "name": "Stmnt_Last",
+                  "openconfig-routing-policy:config": {
+                    "name": "Stmnt_Last"
+                  },
+                  "openconfig-routing-policy:actions": {
+                    "config": {
+                      "policy-result": "ACCEPT_ROUTE"
+                    }
+                  }
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  }
+  ```
+  * Verify that DUT accepted configuration without errors.
+  * Retrive  "test-policy" from device using subscribeRequest once for `/routing-policy/policy-definitions/policy-definition[name="test-policy"]/*`. Compare with policy configured above.
+
+* RT-7.10.3 Policy statement removal
+  * Configure policy "test-policy" and apply using setRequest Replace at `openconfig/routing-policy/`
```

Comment:
This appears to be the same as 7.10.2?  Both contain statements 1,2,6,7,8,10.

If the desire is to delete one, perhaps change this to contain only statements 1,6,7,8,10. ?

Reply:
now is fixed. After removal is 1,6,7,8,10